Makefile

```
@@ -613,24 +593,24 @@ check-packr: bpf/packrd/packed-packr.go
 		false; \
 	fi
 
+SOURCE_DIRS := "./bpf/ ./buildinfo/ ./calc/ ./check-licenses/ ./cmd/ ./config/ ./conntrack/ ./daemon/ ./dataplane/ ./dispatcher/ ./fv/ ./hashutils/ ./ifacemonitor/ ./ip/ ./ipsets/ ./iptables/ ./jitter/ ./k8sfv/ ./labelindex/ ./logutils/ ./markbits/ ./multidict/ ./policysync/ ./proto/ ./routetable/ ./rules/ ./statusrep/ ./stringutils/ ./testutils/ ./throttle/ ./usagerep/ ./versionparse/"
+
 # Run go fmt on all our go files.
 .PHONY: go-fmt goimports fix
 fix go-fmt goimports:
-	$(DOCKER_RUN) $(CALICO_BUILD) sh -c 'glide nv -x | \
-	      grep -v -e "^\\.$$" | \
-	      xargs goimports -w -local github.com/projectcalico/'
+	$(DOCKER_RUN) $(CALICO_BUILD) sh -c 'echo $(SOURCE_DIRS) | xargs goimports -w -local github.com/projectcalico/'
```

This `SOURCE_DIRS` is fragile and creates a maintenance burden, e.g. that we have to remember to update when we add new packages.  How about

```suggestion
	$(DOCKER_RUN) $(CALICO_BUILD) sh -c 'find . -iname "*.go" ! -wholename "./vendor/*" | xargs goimports -w -local github.com/projectcalico/'
```